checker-qual/src/main/java/org/checkerframework/checker/sqlquotes/qual/SqlEvenQuotes.java

```diff
@@ -0,0 +1,22 @@
+package org.checkerframework.checker.sqlquotes.qual;
+
+import java.lang.annotation.*;
+import org.checkerframework.framework.qual.QualifierForLiterals;
+import org.checkerframework.framework.qual.SubtypeOf;
+
+/**
+ * SqlEvenQuotes: Used to denote a String that comprises part of a SQL query and contains either
+ * zero or an even number of unescaped single quotes – i.e., there must be either zero or an even
+ * number of ‘ characters in a SqlEvenQuotes String that are not preceded immediately by a \
+ * character. SqlEvenQuotes Strings are syntactical and safe to be passed to query execution
```

Comment:
We don't know that the string is safe to be passed to a query execution method.
The only thing we know is that this doesn't change whether subsequent concatenations are interpreted as SQL code or as a SQL string.